code/app/src/main/res/layout/activity_main.xml

```diff
@@ -22,4 +22,12 @@
         app:itemTextColor="@drawable/selector"
         app:menu="@menu/menu_navigation" />
 
```

Comment:
File not needed, the button not configured to anything

Reply:
button removed